modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJavaUpdateFileEntryMethodCheck.testjava

```diff
@@ -0,0 +1,40 @@
+package com.liferay.source.formatter.dependencies.upgrade;
+
+public class UpgradeJavaUpdateFileEntryMethodCheck {
+
+	public void method(
+		long userId, long fileEntryId, String sourceFileName, String mimeType,
+		String title, String description, String changeLog,
+		boolean majorVersion, byte[] bytes, ServiceContext serviceContext) {
+
+		_dlAppLocalService.updateFileEntry(
+			userId, fileEntryId, sourceFileName, mimeType, title, description,
+			changeLog, majorVersion, bytes, serviceContext);
+
+		DLAppLocalServiceUtil.updateFileEntry(
+			userId, fileEntryId, sourceFileName, mimeType, title, description,
+			changeLog, majorVersion, bytes, serviceContext);
+
+		DLAppLocalServiceUtil.updateFileEntry(
+			12345, 54321, test, mimeType, title, description, changeLog,
+			majorVersion, bytes, serviceContext);
```

Comment:
@TamyrisTorres, why did you decide to include this method which won't be formatted just here and not in the method below?

Please remove this method or add a similar one in the method above to keep consistency :wink:

Reply:
I need this method to test the message. Now, I added it in the other method as well.